src/wallet/rpcwallet.cpp

```diff
@@ -3716,7 +3716,7 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
                 throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Small transaction amount %s has fee %s that is greater than the default fee %s", FormatMoney(nTotalOut), FormatMoney(nFee), FormatMoney(nDefaultFee)));
             }
         } else {
-            // Check that the user specified fee is sane.
+            // Check that the user specified fee is sane and also greater than the default fee.
```

Comment:
This commit adds to the comment, but @arielgabizon wanted it added to the error message.

Reply:
Right, that was not clear maybe from where I wrote it. Sane here means that the fee is either at most the sum of outputs or at most the default fee. So the code comment should be restored, 'and also greater than the default fee' should be added to the error message in line 3721.

Reply:
Thanks for clarifying, just fixed in 4b8c52c